app/controllers/api/permissions_controller.rb

```diff
@@ -14,10 +14,8 @@ class PermissionsController < ApiController
     def show
       @permissions = if doorkeeper_token.present?
                        doorkeeper_token.scopes.map { |s| :"allow_#{s}" }
-                     elsif current_token.present?
-                       ClientApplication.all_permissions.select { |p| current_token.read_attribute(p) }
                      elsif current_user
-                       ClientApplication.all_permissions
+                       Oauth.scopes.map { |s| :"allow_#{s.name}" }
```

Comment:
Since basic auth is disabled, I would assume that we don’t need this case anymore. Is this still to planned for anther PR, which also removes basic auth usage in unit tests?